src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/TextDifferencing/DiffKind.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.TextDifferencing;
+
+internal enum DiffKind : byte
```

Comment:
is it just habit or does byte make a noticeable difference here?

Reply:
It makes a difference in allocation size, yes.